includes/core/classes/class-venue.php

```diff
@@ -157,8 +157,8 @@ public function register_post_type(): void {
 	 */
 	public static function get_localized_post_type_slug(): string {
 		$switched_locale = switch_to_locale( get_locale() );
-		$slug            = _x( 'venue', 'Post Type Slug', 'gatherpress' );
-		$slug            = sanitize_title( $slug );
+		$slug            = _x( 'Venue', 'Post Type Singular Name', 'gatherpress' );
+		$slug            = sanitize_title( $slug, 'save' );
```

Comment:
setting third argument as second argument. should be `sanitize_title( $slug, '', 'save' );`